Dockerfile.ocp

```diff
@@ -1,16 +1,16 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS builder
 WORKDIR /go/src/github.com/improbable-eng/thanos
 COPY . .
 ENV GOFLAGS="-mod=vendor"
 ENV GO111MODULE=on
 ENV GOPROXY=direct
-RUN make check-git; \
-    if yum install -y prometheus-promu; then \
-        export PROMU=/usr/bin/promu; \
-        promu build; \
-    else \
-        make build; \
-    fi
+# Installing using the upstream Makefile doesn't work with older versions of Git and newer versions of Go.
+# Instead install promu from the upstream tarball.
+# For downstream builds, promu is available from the package repositories.
+RUN if ! yum install -y prometheus-promu; then \
+        curl -s -L https://github.com/prometheus/promu/releases/download/v0.5.0/promu-0.5.0.linux-amd64.tar.gz | tar -xzvf - -C /usr/bin  --strip-components 1 promu-0.5.0.linux-amd64/promu; \
```

Comment:
is this something we need to think about bumping regularly?

Reply:
Not really. We're only using `promu` to build the binary which is a feature that doesn't evolve much (it can only do cross-platform builds, create tarballs, ...).